### ISSUE TITLE
About add korean language

### ISSUE THREAD

**Issue by sincntx** [etherdelta/etherdelta.github.io#85]
I'm a big fan of etherdelta,
so I want to add korean language to etherdelta.
Can I make the pull request about it? :)

**Comment by zackcoburn** [comment]
Absolutely. Actually, if you can work with this file, that would be best:
https://gist.github.com/zackcoburn/954e9ff0a18817ad9e5e6ab17ee5f011

**Comment by spyhole** [comment]
@zackcoburn 
    "en": "Gas price",
    "cn": "气体价格",-->"cn": "燃气价格“ is better.

**Comment by zackcoburn** [comment]
Fixed

**Comment by sincntx** [comment]
My file is here :)

https://gist.github.com/sincntx/eca200d7db0eb22d4554abb60daa6fff

**Comment by zackcoburn** [comment]
Awesome, I will plug this in soon. Great work.
